### PR TITLE
Use multiselect.js to create more user-friendly multi select lists

### PR DESCRIPTION
**Pull request by ejjoman**
Added multiselect.js to project and added initializer code into loader-bootstrap.js and fixed some minor html errors in profile.html.

### CODE REVIEW

File: weblate/static/js/jquery.multi-select.js [hunk 1]
```diff
@@ -0,0 +1,535 @@
...
+
+
+ /* MULTISELECT CLASS DEFINITION
+  * ====================== */
+
+  var MultiSelect = function (element, options) {
+    this.options = options;
+    this.$element = $(element);
+    this.$container = $('<div/>', { 'class': "ms-container" });
+    this.$selectableContainer = $('<div/>', { 'class': 'ms-selectable' });
```
Comment (nijel) [inline weblate/static/js/jquery.multi-select.js]: Codacy Issue found: [jQuery selector "<div/>" previously looked up. First appearance on line 23.](https://www.codacy.com/app/michal_2/weblate/file/1228419940/issues/source?bid=2038971&fileBranchId=2932579#l24)

File: weblate/static/js/jquery.multi-select.js [hunk 2]
```diff
@@ -0,0 +1,535 @@
...
+      var that = this;
+
+      if (options.value !== undefined && options.value !== null){
+        options = [options];
+      } 
+      $.each(options, function(index, option){
+        if (option.value !== undefined && option.value !== null &&
+            that.$element.find("option[value='"+option.value+"']").length === 0){
+          var $option = $('<option value="'+option.value+'">'+option.text+'</option>'),
+              index = parseInt((typeof option.index === 'undefined' ? that.$element.children().length : option.index)),
```
Comment (nijel) [inline weblate/static/js/jquery.multi-select.js]: Codacy Issue found: [Variable 'index' is already declared in line 180.](https://www.codacy.com/app/michal_2/weblate/file/1228419940/issues/source?bid=2038971&fileBranchId=2932579#l184)

File: weblate/static/js/jquery.multi-select.js [hunk 3]
```diff
@@ -0,0 +1,535 @@
...
+          }
+        }
+      } else if (direction === -1){ // UP
+
+        $nextElem = $currElem.prevAll(this.elemsSelector).first();
+        if ($nextElem.length === 0){
+          var $optgroupUl = $currElem.parent();
+
+          if ($optgroupUl.hasClass('ms-optgroup')){
+            var $optgroupLi = $optgroupUl.parent(),
```
Comment (nijel) [inline weblate/static/js/jquery.multi-select.js]: Codacy Issue found: [Variable '$optgroupLi' is already declared in line 258.](https://www.codacy.com/app/michal_2/weblate/file/1228419940/issues/source?bid=2038971&fileBranchId=2932579#l277)

File: weblate/static/js/jquery.multi-select.js (hunk 2 above)
Comment (nijel) [inline weblate/static/js/jquery.multi-select.js]: Codacy Issue found: [You should specify the radix of the mentioned string in parseInt calls.](https://www.codacy.com/app/michal_2/weblate/file/1228419940/issues/source?bid=2038971&fileBranchId=2932579#l184)

File: weblate/static/js/jquery.multi-select.js [hunk 4]
```diff
@@ -0,0 +1,535 @@
...
+            }
+          } else {
+            $nextElem = $elems.first();
+          }
+        }
+      } else if (direction === -1){ // UP
+
+        $nextElem = $currElem.prevAll(this.elemsSelector).first();
+        if ($nextElem.length === 0){
+          var $optgroupUl = $currElem.parent();
```
Comment (nijel) [inline weblate/static/js/jquery.multi-select.js]: Codacy Issue found: [Variable '$optgroupUl' is already declared in line 255.](https://www.codacy.com/app/michal_2/weblate/file/1228419940/issues/source?bid=2038971&fileBranchId=2932579#l274)

File: weblate/static/js/jquery.multi-select.js [hunk 5]
```diff
@@ -0,0 +1,535 @@
...
+            optgroupId = that.sanitize(optgroupLabel),
+            $selectableOptgroup = that.$selectableUl.find('#optgroup-selectable-'+optgroupId),
+            $selectionOptgroup = that.$selectionUl.find('#optgroup-selection-'+optgroupId);
+
+        if ($selectableOptgroup.length === 0){
+          var optgroupContainerTpl = '<li class="ms-optgroup-container"></li>',
+              optgroupTpl = '<ul class="ms-optgroup"><li class="ms-optgroup-label"><span>'+optgroupLabel+'</span></li></ul>';
+
+          $selectableOptgroup = $(optgroupContainerTpl);
+          $selectionOptgroup = $(optgroupContainerTpl);
```
Comment (nijel) [inline weblate/static/js/jquery.multi-select.js]: Codacy Issue found: [jQuery selector "optgroupContainerTpl" previously looked up. First appearance on line 144.](https://www.codacy.com/app/michal_2/weblate/file/1228419940/issues/source?bid=2038971&fileBranchId=2932579#l145)

File: weblate/static/js/jquery.multi-select.js [hunk 6]
```diff
@@ -0,0 +1,535 @@
...
+
+ /* MULTISELECT CLASS DEFINITION
+  * ====================== */
+
+  var MultiSelect = function (element, options) {
+    this.options = options;
+    this.$element = $(element);
+    this.$container = $('<div/>', { 'class': "ms-container" });
+    this.$selectableContainer = $('<div/>', { 'class': 'ms-selectable' });
+    this.$selectionContainer = $('<div/>', { 'class': 'ms-selection' });
```
Comment (nijel) [inline weblate/static/js/jquery.multi-select.js]: Codacy Issue found: [jQuery selector "<div/>" previously looked up. First appearance on line 23.](https://www.codacy.com/app/michal_2/weblate/file/1228419940/issues/source?bid=2038971&fileBranchId=2932579#l25)

File: weblate/static/js/jquery.multi-select.js [hunk 7]
```diff
@@ -0,0 +1,535 @@
...
+        if ($selectableOptgroup.length === 0){
+          var optgroupContainerTpl = '<li class="ms-optgroup-container"></li>',
+              optgroupTpl = '<ul class="ms-optgroup"><li class="ms-optgroup-label"><span>'+optgroupLabel+'</span></li></ul>';
+
+          $selectableOptgroup = $(optgroupContainerTpl);
+          $selectionOptgroup = $(optgroupContainerTpl);
+          $selectableOptgroup.attr('id', 'optgroup-selectable-'+optgroupId);
+          $selectionOptgroup.attr('id', 'optgroup-selection-'+optgroupId);
+          $selectableOptgroup.append($(optgroupTpl));
+          $selectionOptgroup.append($(optgroupTpl));
```
Comment (nijel) [inline weblate/static/js/jquery.multi-select.js]: Codacy Issue found: [jQuery selector "optgroupTpl" previously looked up. First appearance on line 148.](https://www.codacy.com/app/michal_2/weblate/file/1228419940/issues/source?bid=2038971&fileBranchId=2932579#l149)

File: weblate/static/js/jquery.multi-select.js [hunk 8]
```diff
@@ -0,0 +1,535 @@
...
+  * ====================== */
+
+  var MultiSelect = function (element, options) {
+    this.options = options;
+    this.$element = $(element);
+    this.$container = $('<div/>', { 'class': "ms-container" });
+    this.$selectableContainer = $('<div/>', { 'class': 'ms-selectable' });
+    this.$selectionContainer = $('<div/>', { 'class': 'ms-selection' });
+    this.$selectableUl = $('<ul/>', { 'class': "ms-list", 'tabindex' : '-1' });
+    this.$selectionUl = $('<ul/>', { 'class': "ms-list", 'tabindex' : '-1' });
```
Comment (nijel) [inline weblate/static/js/jquery.multi-select.js]: Codacy Issue found: [jQuery selector "<ul/>" previously looked up. First appearance on line 26.](https://www.codacy.com/app/michal_2/weblate/file/1228419940/issues/source?bid=2038971&fileBranchId=2932579#l27)

File: weblate/static/js/jquery.multi-select.js [hunk 9]
```diff
@@ -0,0 +1,535 @@
...
+            $selectionOptgroup = that.$selectionUl.find('#optgroup-selection-'+optgroupId);
+
+        if ($selectableOptgroup.length === 0){
+          var optgroupContainerTpl = '<li class="ms-optgroup-container"></li>',
+              optgroupTpl = '<ul class="ms-optgroup"><li class="ms-optgroup-label"><span>'+optgroupLabel+'</span></li></ul>';
+
+          $selectableOptgroup = $(optgroupContainerTpl);
+          $selectionOptgroup = $(optgroupContainerTpl);
+          $selectableOptgroup.attr('id', 'optgroup-selectable-'+optgroupId);
+          $selectionOptgroup.attr('id', 'optgroup-selection-'+optgroupId);
```
Comment (nijel) [inline weblate/static/js/jquery.multi-select.js]: Codacy Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/michal_2/weblate/file/1228419940/issues/source?bid=2038971&fileBranchId=2932579#l147)

**Comment by nijel** [comment]
Thanks, I've merged with few minor adjustments (5607ab5be525bf895acec6db3b394a948d0fa008 and f28a4114ad4c185f902e8f9c2353ebdb1e3df80a).

**Comment by ejjoman** [comment]
Thank you for merging this. I thought, I needed to fix those Issues before you'll merge this... :)

**Comment by nijel** [comment]
I don't really want to fix javascript issues in all third party libraries :-).

**Comment by Rvanlaak** [comment]
Nice! :+1:

**Pull request merged**